science/healpix/Portfile

```diff
@@ -46,13 +46,17 @@ if {${name} eq ${subport}} {
 }
 
 subport ${name}-java {
+    PortGroup       java 1.0
+
     description         Java language implementation of HEALPix
     long_description    ${long_description} This is the ${description}.
 
     supported_archs noarch
 
-    depends_build   bin:javac:jikes bin:jar:jikes bin:ant:apache-ant
-    depends_lib     bin:java:kaffe
+    java.version    1.4+
```

Comment:
What should the required Java version be? I briefly looked on Healpix website and in the source distribution, and did not find information on what versions are supported. Considering `jikes` was set as the fallback compiler, I imagine much older versions of Java are usable/required.

Reply:
Personally I would not consider anything older than 1.8, even if older ones techncailly could work, as this is currently the older support version upstream.

Reply:
@chrstphrchvz ping. Do you agree to up the minimum java version ?

Reply:
I would rather get input from someone knowledgeable/authoritative before moving forward with this; I have posted to the [healpix-support mailing list](https://sourceforge.net/p/healpix/mailman/healpix-support/) to try finding out. I also don't know whether this port is needed by PowerPC users that might be stuck with e.g. Java 6 or earlier (I have not found evidence that MacPorts' `openjdk8` builds on PowerPC).